curriculum/Chapter-2-Agile-Team-Collaboration.md

```diff
@@ -0,0 +1,140 @@
+Daily scrums improve communications, promote quick decision-making, and consequently eliminate the need for other meetings.
+
+After the completion of each Sprint, the scrum team holds a Sprint retrospective to assess the work done. The team discusses what went well during the Sprint, problems encountered, and how these problems were (or can be) solved. The outcome of a retrospective are action items to improve the effectiveness of the Sprint, which can be incoroprated as soon as the next Sprint cycle. The purpose of these meetings are to increase the efficiency and quality of the Sprint cycle, reinforcing Agile principles like iterative and continuous improvement.
+
+It is important not to conflate the topics of Agile and scrum. Agile set of guiding principles aimed at efficiently delviering value to the end user, and scrum is a framework to apply these principles and ship software to prod. While scrum is an important framework that provides value to balanced teams, it is important to ensure your team is operating in a manner that can be considered "Agile".
+
+## Types of Work
+
+Agile development categorizes work into distinct types, each serving a unique role in delivering user value efficiently. Understanding these categories helps teams prioritize tasks, manage workloads, and maintain a user-focused approach throughout the product lifecycle.
+
+### Discovery and Framing
+
+This initial phase involves researching and defining what to build next, focusing on understanding user needs, market trends, and potential impacts on the product. Effective discovery and framing ensure that the team invests in work that maximizes user and business value.
+
+### Feature Development
+
+Features are new capabilities or enhancements added to a product to meet user needs or improve user experience. Focusing on features allows teams to directly address user requests or market demands, ensuring the product evolves in a way that adds real value.
+
+### Bug Fixes
+
+Addressing bugs is crucial for maintaining the quality and reliability of a product. Quick responses to bug reports reflect a commitment to user satisfaction and product stability, key aspects of maintaining trust and value in users' eyes.
+
+### Tech Debt
+
+Tech debt encompasses code or design elements that, while functional, are inefficient or potentially problematic. Prioritizing tech debt reduction can prevent future bugs, improve performance, and facilitate easier feature development, indirectly enhancing user value.
+
+### Spikes and Mob Learning
+
+Spikes are exploratory tasks aimed at answering specific questions or investigating potential solutions. Mob learning can be applied to spikes, where the team collaborates to solve complex problems or learn new technologies, ultimately speeding up development and enhancing product quality.
+
+## Delivery and Feedback
+
+TODO: Cover project-level deadlines and the importance of realistic estimations to ensure timely delivery.
```

Comment:
Reframe this to testing the app with users, collecting feedback, and going through more D&F or mobius loop concepts.

Delivery is different for each project/product.